CONTRIBUTING.md

```diff
@@ -66,10 +66,20 @@ Instead, write one sentence per line, as this makes it easier to review changes.
 
    An existing example is [libgnunetchat](https://github.com/ngi-nix/ngipkgs/blob/main/pkgs/by-name/libgnunetchat/package.nix).
 
-1. To add a NixOS service module, start by creating a `default.nix` file in the directory `projects/some-project` where `some-project` is the project name corresponding to the last URL component in the [NLnet project listing](https://nlnet.nl/project/).
+1. When contributing to a project, start by checking if it has an entry in `projects/some-project`.
+   If the entry does not exist, you should copy the project template and edit it with relevant details:
```

Comment:
```suggestion
   If the entry does not exist, copy the project template and edit it with relevant details:
```

Always use imperative in instructions. It's fewer words and unambiguous. If you write "you should" it sounds like there may be other options.